### ISSUE TITLE
ALL - cleanup process

### ISSUE THREAD

**Issue by despiegk** [threefoldfoundation/home#156]
- everything we want to do in this sprint (271) needs to be on milestone now, rest on next or later
- for main wiki use wiki_271 as project (project defined on the wiki itself), move away from your main project, will give you less clutter
- on main wiki: go over all issues, make sure whatever we need to do now is linked to wiki_271 and on now
- make sure we don't abuse labels: critical means we do today, we don't go to sleep, major means we do asap < 1-2 working day max, if not then don't flag it, use your kanban to organize
- create more high level stories to organize your work, don't put every little bug in your main project, keep the main project more clean

**Comment by despiegk** [comment]
aim is to know what we need to do over next days, right now its not clear

**Comment by despiegk** [comment]
note here I did use critical because this needs to be done NOW

**Comment by Sacha96** [comment]
On our side (team promo) all issues that could have been done are cleared. We also cleaned a max of issues that were outdated/ already done. Our sprint is organized and we know the priority for this week. Once Team web is done we can close this issue.

**Comment by Sacha96** [comment]
Looking forward to rock this week 💯

**Comment by Vilnite** [comment]
Sprint web_2.7.1 deadline was yesterday so I'm closing it.

It's clear that for web presence weekly releases as for now are too long. 
Too many changes are needed/requested right away, where waiting till following Sunday is just not an option.

Therefore we will stick to weekly sprints but stories could determine when each of it goes out on production. As suggested per Kristof we keep story issues on main project and smaller issues supporting stories are linked to story issues and remains on team projects for execution.

For example, we are starting web_2.8 today, deadline 11th April:
- threefold_io needs to be released asap, so we will finish related issues and release it today (5th April)
- farming has to be updated and can be released by the end of the week (11th April)
- wiki release is set as critical so I believe it needs to happen today (5th April)
etc.

We already use Milestones where current sprint is marked with: Now, next sprints with: Next or Later
Labels is determining priority of each issue (critical - needs to be done NOW, major - done in 1-2 days, minor - done in a week)

**Comment by Vilnite** [comment]
Improvements asked for process wise:
- [ ] the sprints are open WAY too long
- [ ] there are no stories
- [ ] there are no branches to review
- [ ] there are no stakeholder meetings
- [ ] the process not followed around milestone markers
- [ ] the process not followed around making sure we know which release will go out by when
- [ ] no escalation

**Comment by Sacha96** [comment]
The token page of threefold.io and farming website needs to be the main prior and released before Casper announcement on Thursday if we can prioritize as such! According to going through websites, the other main changes are navbars and footers and interlinking.

**Comment by Vilnite** [comment]
Please see stories on current project.
Aligned with Sacha on prio's and process going forward.
Will update and as we go.
